raiden/tests/integration/api/test_pythonapi_regression.py

```diff
@@ -33,26 +33,29 @@ def test_close_regression(raiden_network, deposit, token_addresses):
     amount = PaymentAmount(10)
     identifier = PaymentID(42)
     secret, secrethash = factories.make_secret_with_hash()
-    assert api1.transfer_and_wait(
-        registry_address=registry_address,
-        token_address=token_address,
-        amount=amount,
-        target=TargetAddress(api2.address),
-        identifier=identifier,
-        secret=secret,
-        transfer_timeout=10,
-    )
-    exception = ValueError("Waiting for transfer received success in the WAL timed out")
-    with gevent.Timeout(seconds=5, exception=exception):
+    timeout = block_offset_timeout(app1.raiden)
```

Comment:
Note: This timeout may under shoot by a few blocks, this could happen because the context manager is created before the transfer is started. When the transfer is actually started a few blocks could have passed by.

This should not be a huge problem since we expect the transfer in the test to finish readily.